Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+def ORG = "mayadataio"
+def REPO = "openebs-upgrade"
+
+pipeline {
+    agent any
+    stages {
+        stage('Build Image') {
+            steps {
+                script {
+                  GIT_SHA = sh(
+                            returnStdout: true,
+                            script: "git log -n 1 --pretty=format:'%h'"
+                            ).trim()
+                  echo "Building Go Code"
+                  sh "docker build -t ${ORG}/${REPO}:ci-${GIT_SHA} ."
+                }
+            }
+        }
+        stage('Push Image') {
+            steps {
+                script {
+		             withCredentials([usernamePassword( credentialsId: 'docke_cred', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+                            if (env.BRANCH_NAME == 'master')  {
+                               echo "Pushing the image with the tag..."
+                               sh "docker login -u${USERNAME} -p${PASSWORD} "
+			                         sh "docker push mayadataio/openebs-upgrade:${tag}"
+                            } else {
+			                   echo "WARNING: Not pushing Image"
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
Pipeline should be closed in last but it is cloning here. Remove this extra bracket

Reply:
Done